ad-code-manager.php

```diff
@@ -614,7 +615,7 @@ function action_load_ad_code_manager() {
 		switch ( $this->wp_list_table->current_action() ) {
 		case 'delete':
 			check_admin_referer( 'acm-bulk-action', 'bulk-action-nonce' );
-			$ad_code_ids = array_map( 'intval', $_REQUEST['ad-codes'] );
+			$ad_code_ids = array_map( 'intval', sanitize_key( $_REQUEST['ad-codes'] ) ); // Input var okay.
```

Comment:
What's in $_REQUEST['ad-codes'] it looks like it's an array ans we're trying to apply sanitize_key() to it. I suspect that won't work. It also seems unnecessary if we're doing intval() on all the array values.